src/main/java/Duke.java

```diff
@@ -1,10 +1,287 @@
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String div = "____________________________________________________________\n";
+
+        class Task {
+            protected String name;
+            protected boolean isDone = false;
+
+            public Task(String name) {
+                this.name = name;
+            }
+
+            public void mark(){
+                isDone = true;
+            }
+
+            public void unmark(){
+                isDone = false;
+            }
+
+            public String printTask() {
+                return (isDone) ? "[X] " + name + "\n" : "[ ] " + name + "\n";
+            }
+
+            public void addToFile() {
+                try {
+                    FileWriter Writer
+                            = new FileWriter("data/data.txt", true);
```

Comment:
would it be better to "abstract" away the file path by using a constant declaration instead? It would be painful to refactor if I had to change the file path.